### PR TITLE
refactor: cleanup after logs.opencraft.com

### PR DESCRIPTION
**Pull request by gabor-boros**
Remove filebeat from the stack as we dismantle logs.opencraft.com and replaced filebeat with fluent-bit.

### CODE REVIEW

File: playbooks/appserver.yml [hunk 1]
```diff
@@ -9,10 +9,6 @@
       when: consul_servers | length > 0
     - role: node-exporter
       when: NODE_EXPORTER_PASSWORD is not none
-    - role: filebeat
-      # Appservers may have some Elastic stuff already installed on them.
-      filebeat_elastic_apt_pin: 200
-      when: filebeat_logstash_hosts | length > 0
```
Comment (mtyaka) [inline playbooks/appserver.yml:15]: I don't think we use the `appserver.yml` book anymore (it was only used by Ocim), so we could remove the entire playbook.

File: playbooks/deploy-all.yml [hunk 2]
```diff
@@ -3,7 +3,6 @@
 - import_playbook: accounting.yml
 - import_playbook: backup-pruner.yml
 - import_playbook: elasticsearch.yml
-- import_playbook: elk.yml
```
Comment (mtyaka) [inline playbooks/deploy-all.yml:6]: We could also remove the elk playbook from this repo if it's no longer needed.

File: playbooks/appserver.yml (hunk 1 above)
Comment (gabor-boros) [reply playbooks/appserver.yml:15]: No, we don't, but I'd like to keep the PR "focused" as this repo will be archived anyways.

**Review by mtyaka — APPROVED** [review]
:+1:

- [x] I read through the code

**Pull request merged**